### PR TITLE
Replace literals

### PR DESCRIPTION
**Pull request by SvenRosvall**
This is to make use of the constants in VLCB-defs for self-documentation and for avoiding errors.

### CODE REVIEW

File: src/Configuration.cpp
Comment (MartinDaCosta53) [inline src/Configuration.cpp:1]: Where is the value of EE_HASH_BYTES defined?  It is derived from the number of Node Number bytes plus the number of Event Number bytes and so is always 4.  That is why I took it out of the user sketch and placed it in the library.  No issue with the constant but it needs to be defined in Configuration.h in my view

Comment (SvenRosvall) [reply src/Configuration.cpp:1]: It is defined in Configuration.h. See line 17.

Comment (SvenRosvall) [reply src/Configuration.cpp:1]: Just a note that it was EE_BYTES_PER_EVENT you took out. This is 4 bytes plus the number of event variables.

Comment (MartinDaCosta53) [reply src/Configuration.cpp:1]: Ah yes.  I didn't see the associated change in Configuration.h.  Now I've found it.
Yes, I agree about EE_BYTES_PER_EVENT, but I think that the reason for the the constant being 4 in both cases is the same.

**Pull request merged**